linz-data-importer/metadata.txt

```diff
@@ -6,10 +6,11 @@ qgisMinimumVersion=3.0.0
 qgisMaximumVersion=3.99.0
 description=Import LDS OGC Datasets into QGIS
 version=2.2.0
-author=Land Information New Zealand
+author=Toitū Te Whenua Land Information New Zealand
```

Comment:
Shouldn't there be a slash or some other punctuation between "Toitū Te Whenua" and "Land Information New Zealand"?

Reply:
Apparently not. https://linzsrm.sharepoint.com/sites/LINZ-news/SitePages/news-Your-questions-answered--Toit%C5%AB-Te-Whenua.aspx?web=1#how-do-we-write-our-name-in-full-now